charter.md

```diff
@@ -23,15 +23,15 @@ The WebTransport working group will define new client-server protocols
 or protocol extensions in order to support the development of the
 W3C WebTransport API <https://wicg.github.io/web-transport>.
 
-The WebTransport working group will define a protocol or suite of
-protocols that support a range of simple communication methods.
+The WebTransport working group will define an application-layer protocol or suite of
+application-layer protocols that support a range of simple communication methods.
```

Comment:
What's about e.g. saying ".. define one or multiple bindings to provide the following simple communication methods on top of QUIC, HTTP/3, and/or TLS/TCP"...?

Reply:
I'm personally not a fan of the word "binding" as it is more vague than "application-layer protocol"